MediaBrowser.Providers/TV/SeriesMetadataService.cs

```diff
@@ -67,6 +67,24 @@ protected override void MergeData(MetadataResult<Series> source, MetadataResult<
 
             var sourceItem = source.Item;
             var targetItem = target.Item;
+            var sourceSeasonNames = sourceItem.SeasonNames;
+            var targetSeasonNames = targetItem.SeasonNames;
+
+            if (replaceData || targetSeasonNames is null)
+            {
+                targetItem.SeasonNames = sourceSeasonNames;
+            }
+            else if (!targetSeasonNames.Equals(sourceSeasonNames))
```

Comment:
Is it supposed to be reference equals?

Reply:
.Equals for Collections are somewhat undefined, so it almost always falls back to ReferenceEquals